.github/workflows/build.yml

```diff
@@ -28,12 +28,17 @@ jobs:
       BUILD_PLATFORM: x64
 
     steps:
-    - uses: actions/checkout@629c2de402a417ea7690ca6ce3f33229e27606a5
+    - name: Harden Runner
```

Comment:
Also, the security hardener doesn't do anything but warn on Windows, so I'd personally remove it for now.

Reply:
> Also, the security hardener doesn't do anything but warn on Windows, so I'd personally remove it for now.

JFYI - I am tracking an [issue](https://github.com/step-security/secure-workflows/issues/570) for the tool to only add harden-runner for ubuntu runners.

Reply:
Removed from this PR in windows worksflows